README.md

```diff
@@ -84,14 +84,25 @@ Users must ensure Docker is installed before attempting most scenarios. Please r
   ```
 
 ### ⚙️ Configuration
-- You have to config your GPT model in the `.env`
```

Comment:
The demos requires following ability
- ChatCompletion
- json_mode
- embedding query